src/EventGridExtension/EventGridExtensionConfig.cs

```diff
@@ -34,6 +46,12 @@ public void Initialize(ExtensionConfigContext context)
             Uri url = context.GetWebhookHandler();
             _tracer.Trace(new TraceEvent(System.Diagnostics.TraceLevel.Info, $"registered EventGrid Endpoint = {url}"));
 
+            // use converterManager as a hashTable
+            // also take benefit of identity converter
+            ConverterManager = context.Config.GetService<IConverterManager>();
+            ConverterManager.AddConverter<JObject, string, EventGridTriggerAttribute>((jobject, attr) => jobject.ToString(Formatting.Indented));
+            ConverterManager.AddConverter<JObject, ReferenceOpenType, EventGridTriggerAttribute>((tSrc, tDest) => (input => ((JObject)input).ToObject(tDest)));
```

Comment:
@MikeStall  do you mind take a look at my use of `converters`?

Reply:
This looks right, pending the question about ReferenceOpenType vs. something else.

Reply:
You shouldn't need ConverterMAnager directly here, call the methods on ExtensionConfigContext . IE: 
context.AddOpenConverter(...) 

Since you're using an OpenType, it should be AddOpenConverter, rather than just AddConverter.